sky/shell/platform/ios/framework/Source/FlutterViewController.mm

```diff
@@ -487,6 +487,7 @@ - (void)setMessageListener:(NSObject<FlutterMessageListener>*)listener
        forMessagesWithName:(NSString*)messageName {
   NSAssert(listener, @"The listener must not be null");
   NSAssert(messageName, @"The messageName must not be null");
+  [listener retain];
```

Comment:
scoped_nsprotocol will take care of calling release for us.  It just expects to adopt a reference in its constructor, so we need to retain the object before calling the constructor.